app/models/user.rb

```diff
@@ -1,5 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable,
```

Comment:
@hvillero we should look into implementing `:omniauthable` in here to match our local setup. 

Then for local development we can build out a flow to allow developers to get in. Let's sync on this today